hoomd/operation.py

```diff
@@ -497,6 +306,13 @@ class Operation(_HOOMDBaseObject):
     `Writer`, `Compute`, `Tuner`, and `hoomd.integrate.BaseIntegrator`. All
     HOOMD-blue operations inherit from one of these five base classes. To find
     the purpose of each class see its documentation.
+
+    Note:
```

Comment:
I think you may have misunderstood my comment. I don't think there should be a link to `ARCHITECTURE.md` here, but there should be a brief description that all `Operation` instances are picklable. Without this mention in the documentation, the change log is the only place that users will be able to see pickling mentioned.

Reply:
I did misunderstand... I will fix that.